### PR TITLE
Exams by Anastasia Dvoretska | IM-21

### PR DESCRIPTION
**Pull request by xastia**

### CODE REVIEW

**Comment by Kabadzh0b** [comment]
Perfect code

File: Tasks/day.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
 // Get day number
+'use strict';
 
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const Week = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
Comment (tshemsedinov) [inline Tasks/day.js:4]:
```suggestion
const DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
or `WEEK` or `WEEK_DAYS` or `DAYS_OF_WEEK`

File: Tasks/longest.js [hunk 2]
```diff
@@ -1,28 +1,16 @@
...
-        } else {
-          i = -1;
-        }
-      }
+const longestWord = (line = []) => {
+  let maxLength = -1;
+  let longest = undefined;
+  for (const word of line) {
+    if (word.length > maxLength) {
+      maxLength = word.length;
```
Comment (tshemsedinov) [inline Tasks/longest.js:9]:
```suggestion
    const { length } = word;
    if (length > maxLength) {
      maxLength = length;
```

**Review by tshemsedinov — APPROVED** [review]
LGTM (looks good to me)

**Pull request closed without merging**